pkg/controllers/builder.go

```diff
@@ -356,6 +358,8 @@ func makeServerContainer(vdb *vapi.VerticaDB, sc *vapi.Subcluster) corev1.Contai
 		},
 		VolumeMounts: buildVolumeMounts(vdb),
 	}
+	cnt.Env = append(cnt.Env, getEnvVars(vdb)...)
```

Comment:
What happens if we have two environment variables with the same name?  We have some ones that already have (e.g. POD_IP).  If we had an annotation called `pod-ip`, what happens?

Reply:
Good remark. It will overwrite in this case the previous POD_IP value and that is not something we want. So we probably need to disapprove any annotation that translates to one of the internally defined env vars (POD_IP, DATA_PATH, DEPOT_PATH).

Reply:
Could we reorder things so that we add POD_IP etc. _after_ adding the annotations?

Reply:
Yes that way if any of these env vars is set in the annotations, its value will be overwritten by the correct one.